src/js/Controllers/TTSController.js

```diff
@@ -135,33 +70,61 @@ class TTSController {
     }
 
     stopSpeak(stopSpeakingID) {
-        const speakID = this.speakID;
-        this.speakID = null;
-        if (this.currentlyPlaying === "FILE") {
-            this.audioPlayer.onended = null;
-            this.audioPlayer.pause();
-            this.audioPlayer.src = "";
-        } else if (this.currentlyPlaying === "TEXT") {
-            window.speechSynthesis.pause();
-            window.speechSynthesis.cancel();
+        var file = null;
+        while (file = this.filePlaylist.shift()) {
```

Comment:
This is causing a warning when building the app

Reply:
fixed warning in https://github.com/smartdevicelink/generic_hmi/pull/406/commits/d15c407092b3d39f0bdb0869f176896fe7a46b7e